lib/crypto/c_src/otp_test_engine.c

```diff
@@ -37,7 +37,7 @@
     PACKED_OPENSSL_VERSION(MAJ,MIN,FIX,('a'-1))
 
 #if OPENSSL_VERSION_NUMBER < PACKED_OPENSSL_VERSION_PLAIN(1,1,0) \
-    || LIBRESSL_VERSION_NUMBER < 0x3050000fL
```

Comment:
That's actually not needed due to a quirk/feature in the preprocessor causing undefined macros to be replaced with 0 in #if-conditions.

I thought of maybe (ab)using that to define all-in-one macros like this

```
#ifdef LIBRESSL_VERSION_NUMBER
#  define HAS_LIBRESSL LIBRESSL_VERSION_NUMBER
#else  /* assume OpenSSL */
#  define HAS_OPENSSL OPENSSL_VERSION_NUMBER
#endif
```
in order to write simpler conditions like
```
#if HAS_OPENSSL > PACKED_VERSION_NUMBER(1,1,0) || HAS_LIBRESSL > 0x3050000f
#  define HAS_SOME_FEATURE
#endif
```